src/i18n/translations/pl.json

```
@@ -0,0 +1,74 @@
+{
+  "dash": {
+    "header": {
+      "your_account_heading": "Twoje konto",
+      "account": "Konto",
+      "settings": "Ustawienia",
+      "log_out": "Wyloguj się"
+    },
+    "sidebar": {
+      "logo_by": "od",
+      "tournament": {
+        "catname": "turniej",
+        "overview": "Panel główny",
+        "ladder": "Drabinka turniejowa",
+        "standings": "Rankingi"
+      },
+      "tournament_data": {
+        "catname": "informacje o turnieju",
+        "competing_teams": "Drużyny",
+        "debate_motions": "Tezy debat",
+        "staff_judges_bias": "Organizatorzy, sędziowie i powiązania",
+        "physical_infrastructure": "Miejsca",
+        "event_branding": "Szata graficzna"
+      },
+      "shareable": {
+        "catname": "media",
+        "image_generation": "Generowanie grafik"
+      },
+      "organisational": {
+        "catname": "organizacja",
+        "audit_log": "Dziennik zdarzeń",
+        "users": "Użytkownicy",
+        "tournament_settings": "Ustawienia turnieju"
+      }
+    }
+  },
+  "login": {
+    "msg_handle_empty": "Nick nie może być pusty.",
+    "msg_passwd_empty": "Hasło nie może być puste.",
+    "msg_invalid_credentials": "Błędne dane logowania",
+    "handle_label": "Nick",
+    "handle_aria_label": "Twój nick",
+    "handle_placeholder": "nick",
```

This will make it inline with the password placeholder semantically and visually, as just 'nick' is disturbingly short in comparison.
```suggestion
    "handle_placeholder": "twój_nick",
```